opt/build/libtorch/repack.sh

```diff
@@ -24,8 +24,13 @@ else
 	fi
 fi
 
-OS=$(python3 $ROOT/opt/readies/bin/platform --os)
-ARCH=$(python3 $ROOT/opt/readies/bin/platform --arch)
+# set them internally or externally
+if [ -z ${OS} ]; then
+    OS=$(python3 $ROOT/opt/readies/bin/platform --os)
+fi
+if [ -z ${ARCH} ]; then
+    ARCH=$(python3 $ROOT/opt/readies/bin/platform --arch)
+fi
```

Comment:
part of readis repo, no?

Reply:
So this allows us to use readies if it's there, if not we have an _approximation_ based on the outputs we received from the script. I'd say this isn't ideal - but it's a good enough first step.